modules/apps/portal-search-solr8/portal-search-solr8-impl/src/main/resources/META-INF/resources/schema.xml

```diff
@@ -592,7 +772,9 @@
 		<dynamicField docValues="true" indexed="true" multiValued="false" name="*Number_sortable" stored="true" type="double" />
 		<dynamicField docValues="true" indexed="true" multiValued="false" name="*String_sortable" stored="true" type="string" />
 		<dynamicField docValues="true" indexed="true" multiValued="false" name="*_sortable" stored="true" type="string" />
+		<dynamicField indexed="true" multiValued="false" name="*_String_sortable.keyword_lowercase" stored="false" type="string_keyword_lowercase" />
```

Comment:
i think you can delete this line. *_sortable will also catch *_String_sortable and the mapping is the same

Reply:
That makes sense to me, I'll try it out. 

As a side note, would that also mean this existing line is also unneccessary?
```
<dynamicField docValues="true" indexed="true" multiValued="false" name="*String_sortable" stored="true" type="string" />
```